CONTRIBUTORS.md

```diff
@@ -0,0 +1,33 @@
+# Contributors
+
+- Alexander Knipping ([@obitech](https://github.com/obitech))
+- Amauri Bizerra ([@AmauriAires](https://github.com/AmauriAires))
+- anirudh-modi ([@anirudh-modi](https://github.com/anirudh-modi))
+- ENT8R ([@ENT8R](https://github.com/ENT8R))
+- Flo Becker ([@beckerei](https://github.com/beckerei))
+- Josh Cooper ([@JoshCooperr](https://github.com/JoshCooperr))
+- Linus Brogan ([@linusaurusrex](https://github.com/linusaurusrex))
+- Metta Ong ([@ongspxm](https://github.com/ongspxm))
+- Michael Corwin ([@mcorwin22](https://github.com/mcorwin22))
+- OogieBoogieInJSON ([@OogieBoogieInJSON](https://github.com/OogieBoogieInJSON))
+- redagavin ([@redagavin](https://github.com/redagavin))
+- xlomonosvx ([@xlomonosvx](https://github.com/xlomonosvx))
+
+## Translations
```

Comment:
> Changed the heading from Translators to Translations so the links

Ah no, sorry. That's not what I meant.

Actually, I just had a look and in all current translations there is not even such a link. (the one you possibly mean goes to the `CONTRIBUTING.md` file, that is unrelated to this one, here.) This is then only relevant for later translations.

So sorry, I guess "translators" is really the better term here, this file is about humans, after all… :wink: 
```suggestion
## Translators
```